lnbits/wallets/corelightningrest.py

```diff
@@ -129,7 +129,10 @@ async def create_invoice(
         return InvoiceResponse(True, label, data["bolt11"], None)
 
     async def pay_invoice(self, bolt11: str, fee_limit_msat: int) -> PaymentResponse:
-        invoice = lnbits_bolt11.decode(bolt11)
+        invoice = decode(bolt11)
```

Comment:
- doesn't `decode` now throw `Bolt11Exception` if it fails?
- `try-catch` should be added as in `lnbits/wallets/corelightning.py`

Reply:
done